doc/md/concepts/integrity_hash_file.md

```diff
@@ -0,0 +1,29 @@
+---
+id: hash_file
+slug: /concepts/integrity_hash_file
+title: Integrity Hash File
+---
+
+## Version Migration Files
+Version migration files are files that describe changes to the databases. Usually these files are prefixed with a number - which describe the version of the database change. The framework that runs these files uses these numbers to determine which changes have been applied to the database.
+
+In Ent, you’ll find these files in ent/migrations/ folder.
+In Django, you’ll find these files in project_name/app_name/migrations/ folder.
```

Comment:
Why Ent and Django are mentioned here? Feels redundant to me.

Reply:
Will remove this in a later commit.